src/Uno.Extensions.Reactive/Collections/Tracking/CollectionAnalyzer.cs

```diff
@@ -13,45 +14,29 @@ namespace Uno.Extensions.Collections.Tracking;
 /// </summary>
 internal partial class CollectionAnalyzer
 {
-	protected delegate int IndexOf<in TCollection>(object item, TCollection snapshot, int startIndex);
-	protected delegate bool VersionEqual(object oldItem, object newItem);
+	internal delegate int IndexOf(object item, int startIndex, int count);
+	internal delegate int IndexOf<in TCollection>(object item, TCollection snapshot, int startIndex);
+	internal delegate bool VersionEqual(object oldItem, object newItem);
```

Comment:
```suggestion
	internal delegate int IndexOfHandler(object item, int startIndex, int count);
	internal delegate int IndexOfHandler<in TCollection>(object item, TCollection snapshot, int startIndex);
	internal delegate bool VersionEqualHandler(object oldItem, object newItem);
```

Reply:
I takes note of this, but I actually already changed those in my pending work, I'll rename directly on my branch.